Multisig/UI/WalletConnectV2/WalletConnectManager.swift

```diff
@@ -15,90 +15,98 @@ import WalletConnectUtils
 import WalletConnectRouter
 import Web3
 import Web3Wallet
+import UIKit
 
 class WalletConnectManager {
     static let shared = WalletConnectManager()
-    
+
     private var publishers = [AnyCancellable]()
     private var dappConnectedTrackingEvent: TrackingEvent?
 
     private let metadata = AppMetadata(
-        name: Bundle.main.displayName,
-        description: "The most trusted platform to manage digital assets on Ethereum",
-        url: App.configuration.services.webAppURL.absoluteString,
-        icons: ["https://app.safe.global/favicons/mstile-150x150.png",
-                "https://app.safe.global/favicons/logo_120x120.png"])
+            name: Bundle.main.displayName,
+            description: "The most trusted platform to manage digital assets on Ethereum",
+            url: App.configuration.services.webAppURL.absoluteString,
+            icons: ["https://app.safe.global/favicons/mstile-150x150.png",
+                    "https://app.safe.global/favicons/logo_120x120.png"])
 
-    private init() { }
+    private init() {
+    }
 
     func config() {
         Networking.configure(projectId: App.configuration.walletConnect.walletConnectProjectId,
-                             socketFactory: SocketFactory())
+                socketFactory: NativeSocketFactory())
```

Comment:
I would like to use Native WebSocket implementation for debugging wire content. And if it works reliably get rid of Starscream in a future PR